base/set.jl

```diff
@@ -145,6 +145,23 @@ setdiff!(s::Set, xs) = (for x=xs; delete!(s,x); end; s)
 <( l::Set, r::Set) = (length(l) < length(r)) && (l <= r)
 <=(l::Set, r::Set) = issubset(l, r)
 
+"""
+    issubset(a, b)
+    ⊆(a,b) -> Bool
+    ⊈(a,b) -> Bool
+    ⊊(a,b) -> Bool
```

Comment:
or given their own separate docstrings that reference this one. probably better as followup that could also fix `in`